anvil/src/drawing.rs

```diff
@@ -169,7 +169,7 @@ where
         } else {
             3
         };
-        Rectangle::from_loc_and_size((0, 0), (24 * digits, 35)).to_f64()
+        Rectangle::new((0, 0), (24 * digits, 35)).to_f64()
```

Comment:
There is actually a `rect` method in euclid to make this even shorter

Reply:
Right. I guess the cases with `origin` `(0, 0)` should just be equivalent to `Rect::from_size`.